### PR TITLE
[zh] Sync Comprehensive Network Security at Splunk blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync Comprehensive Network Security at Splunk into Chinese:
```
content/zh/blog/2023/network-security-splunk/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: "基于 Splunk 的全面网络安全"
+description: "使用 Istio 从 Layer 3 到 Layer 7 的安全性及更多内容。"
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:3]:
```suggestion
description: "Istio 搭配其他工具确保 3 到 7 层网络安全。"
```
layer 可以翻出来吧

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+publishdate: 2023-04-03
+attribution: "Bernard Van De Walle (Splunk), Mitch Connors (Aviatrix)"
+keywords: [Istio,Security,Use Case]
+---
+
+市面上可用的网络安全保障工具种类非常丰富，也很容易找到这些独立工具的相关教程和演示，
+可以为流量添加身份识别、策略和可观测性能力来使您的网络更加安全。
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:10]:
```suggestion
这些工具可以为流量添加身份标识、策略和可观测性能力来使您的网络更加安全。
```

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+使用到的工具如何相互作用以及其中一些可改进的内容。在 Splunk
+中使用多种工具来为其网络提供安全保障，包括：
+
+* AWS 各项功能
+* Kubernetes
+* Istio
+* Envoy
+* Aviatrix
+
+## 关于 Splunk 的使用案例 {#about-splunks-use-case}
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:26]:
```suggestion
## 关于 Splunk 的使用场景 {#about-splunks-use-case}
```

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+Splunk 是一家提供用于收集、分析以及各类数据可视化平台的科技公司。
+通过网页风格界面对机器生成的大量数据进行搜索、监控以及分析。
+Splunk Cloud 是一项将 Splunk 内部基础架构迁移到云原生架构的计划。
+如今，Splunk Cloud 由超过 35 个位于 AWS 和 GCP 遍布全球的完整集群副本组成。
+
+## 为 Layer 3/4 提供安全保障：AWS、Aviatrix 和 Kubernetes {#securing-layer-34-aws-aviatrix-and-kubernetes}
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:33]:
```suggestion
## AWS、Aviatrix 和 Kubernetes 为 3/4 层提供安全保障 {#securing-layer-34-aws-aviatrix-and-kubernetes}
```
这样直接点呢

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+在 Splunk Cloud 中，我们使用一种被称为“cookie cutter VPCs”的模式，
+为每个集群配备其自身的 VPC 网络，并且使 Pod 和节点 IP 具有相同的私有子网，
+另外还提供进出公共互联网的公共子网，以及一个用于集群之间流量的内部子网。
+这使来自不同集群的 Pod 和节点完全隔离，同时允许集群外部流量在公共和内部子网中强制执行特定规则。
+此外，这种模式避免了在管理多个集群时 RFC 1918 专用 IP 耗尽的可能性。
+
+在每个 VPC 中，Network ACL 和 Security Group 被设置为将连接限制在需要范围内。
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:41]:
```suggestion
在每个 VPC 中，Network ACL 和 Security Group 被设置为将连通性限制在绝对必需的范围内。
```

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+例如，我们将公共连接限制在 Ingress 节点中（会部署 Envoy 入口网关）。
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:42]:
```suggestion
例如，我们将公开连通性限制在（将部署 Envoy 入口网关的）Ingress 节点中。
```
公开 or 公共？

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+{{< image width="90%"
+    link="CNCS 2023 - VPC Connectivity 3.png"
+    caption="Splunk 网络安全架构"
+    >}}
+
+Splunk 技术栈中的下一个安全层级则是 Kubernetes 本身。用于验证功能的 Webhook
+将防止某些 K8S 对象的部署，这些对象会允许集群中的不安全流量（通常围绕 NLB 和
+Service）。Splunk 还依赖于 `NetworkPolicies` 来保护和限制 Pod 之间的连接。
+
+## 为 Layer 7 提供安全保障：Istio {#securing-layer-7-istio}
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:55]:
```suggestion
## Istio 为 7 层提供安全保障 {#securing-layer-7-istio}
```

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+
+Splunk 使用 Istio 根据每个请求的详细信息在应用层执行相关策略。
+Istio 也会发送 Telemetry 数据（指标、日志、跟踪），可用于验证请求级安全性。
+
+Istio 注入 Envoy Sidecar 的其中一个主要优势是 Istio
+可以为整个网格提供传输过程中的加密能力，而无需对应用程序进行任何修改。
+应用程序只发送纯文本 HTTP 请求，但 Envoy Sidecar
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:62]:
```suggestion
应用程序只发送明文 HTTP 请求，但 Envoy Sidecar
```

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+拦截流量并实施相互 TLS 加密以防止劫持或篡改。
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:63]:
```suggestion
拦截流量并实施双向 TLS 加密以防止劫持或篡改。
```

File: content/zh/blog/2023/network-security-splunk/index.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+网关由平台团队管理并在 Istio Gateway 命名空间中运行，允许用户插拔，
+但不能被用户修改。Gateway 服务还具备默认情况下强制执行 TLS 证书的能力，
+用于验证的 Webhook 确保服务只能通过自身主机名连接到网关。此外，
+在流量能够对应用程序 Pod 产生影响之前，网关会在入口强制对其执行请求身份验证。
+
+由于 Istio 和相关的 K8S 对象配置起来相对复杂，所以 Splunk
+创建了一个抽象层，它是一个可以为服务配置一切内容的控制器，包括：
+VirtualService、DestinationRule、Gateway、证书等等。
+它会将 DNS 直接写入正确的 NLB 中。它是用于端到端网络部署的一键式解决方案。
+对于更复杂的用例，服务团队仍然可以绕过该抽象实现直接配置这些内容。
```
Comment (windsonsea) [inline content/zh/blog/2023/network-security-splunk/index.md:75]:
```suggestion
对于更复杂的场景，服务团队仍然可以绕过该抽象实现直接配置这些内容。
```

**Review by windsonsea — APPROVED** [review]
Thanks. The zh version looks good to me. 

IMO, the en version can be further improved such as:

- remove spaces from filename
- make the image clearer such as svg

**Comment by wilsonwu** [comment]
> 

Right when I translate, I found lots of more spaces between sentences, I think I can help to fix it later.

**Pull request merged**